src/site/apt/examples/class-relocation.apt.vm

```diff
@@ -70,3 +70,20 @@ Relocating Classes
   into the package <<<org.shaded.plexus.util>>> by moving the corresponding JAR file entries and rewritting the
   affected bytecode. The class <<<Xpp3Dom>>> and some others will remain in their original package.
 
+
+  It's also possible to narrow the pattern with the "include" tag:
```

Comment:
Could you please replace `"` with `<<<` and the closing with `>>>` accordingly.

Reply:
Sure thing! Done.